api/admin_test.go

```diff
@@ -149,6 +150,18 @@ func TestSaveConfig(t *testing.T) {
 	}
 
 	th.App.UpdateConfig(func(cfg *model.Config) { *cfg.TeamSettings.EnableOpenServer = true })
+
+	// Should not be able to modify PluginSettings.EnableUploads
```

Comment:
nit: Break out this test to start the exodus from the monoliths we have today?

Reply:
Yeah, good point. I can break out the api4 test but the api package is going to be removed in a few months so I don't think making changes there is worth it